README.rst

```diff
@@ -74,7 +74,7 @@ Here's a quick summary of resources to help you find your way around:
   tracked separately at https://zephyrprojectsec.atlassian.net.
 * **Zephyr Project Website**: https://zephyrproject.org
 
-.. _Slack Invite: https://tinyurl.com/2vue8666
+.. _Discord Invite: https://discord.gg/Ck7jw53nU2
```

Comment:
Note that in the future, through partnership or Server boost, we will be able to have a custom URL.